### DISCUSSION TITLE
Will deleting file in iCloud delete also this file in my backup?

### DISCUSSION THREAD

**Discussion by mareczek82** [icloud-photos-downloader/icloud_photos_downloader#633] [Q&A]
Hello,

Thank you for this amazing tool, i have one question about syncing as in title: will deleting file in photo app, will delete same file in my Linux backup after sync proces?

**Comment by AndreyNikiforov** [comment]
> will deleting file in photo app, will delete same file in my Linux backup after sync proces?

If you use `--auto-delete` option then all photos removed in iCloud (they are actually moved to "Recently Deleted" folder for 30 days and then purged) will also be deleted on your local drive.

**Reply by mareczek82** [reply]
Ok, thank you. 

It wasn't clear for me, I thought it deletes files in icloud "recently deleted album and was confused why it didn't delete them :)

Thank you again and have a nice day!
